src/status_im/navigation/screens.cljs

```diff
@@ -140,6 +141,10 @@
      :options   options/transparent-screen-options
      :component share-community-channel/view}
 
+    {:name      :share-community
```

Comment:
i believe we want to use :screen/ namespace dunno if we have this documented

Reply:
Fixed.